testData/cases/dielectric_refraction/dielectricRefraction.py

```diff
@@ -0,0 +1,120 @@
+# %%
+import numpy as np
+from numpy.fft import *
+import matplotlib.pyplot as plt
+import scipy.constants
+from skrf.media import Freespace
+from skrf.frequency import Frequency
+
+import sys, os
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../../', 'src_pyWrapper'))
+SEMBA_EXE = '../../../build/bin/semba-fdtd'
+
+from pyWrapper import *
+# %%
+dt = 1e-12
+w0 = 0.1e-8
+t0 = 10*w0
+t = np.arange(0, t0+2*t0, dt)
+f = np.exp( -np.power(t-t0,2)/ w0**2 )
+
+#plt.figure()
+#plt.plot(t,f)
+
+data = np.zeros((len(t), 2))
+data[:,0] = t
+data[:,1] = f
+np.savetxt('gauss.exc', data)
+
+#%%
+fwave = np.sin(w0*t)
+
+data = np.zeros((len(t), 2))
+data[:,0] = t
+data[:,1] = fwave
+np.savetxt('wave.exc', data)
+
+#%%
+fq = fftfreq(len(t))/dt
+F = fft(f)
+#%%
+F = F[fq>=0]
+fq = fq[fq>=0]
+Fmax = np.max(np.abs(F))
+Fcut = Fmax / np.sqrt(2.0)
+idx = (np.abs(np.abs(F) - Fcut)).argmin()
+
+plt.figure()
+plt.plot(fq, np.abs(F),'.-')
+plt.axvline(x = fq[idx], color = 'r')
+plt.xlim(0,5e9)
+plt.ylim(1e-9, 1e6)
+plt.yscale('log')
+
+plt.grid()
+# %%
+fn = 'dielectricRefraction.fdtd.json'
+solver = FDTD(input_filename = fn, path_to_exe=SEMBA_EXE)
+solver.cleanUp()
+solver.run()
+assert solver.hasFinishedSuccessfully()
+
+#%%
+outside = Probe(solver.getSolvedProbeFilenames("outside")[0]) 
+plt.plot(outside.df['time'],  outside.df['field'], label='outside')
```

Comment:
In my case I need to add .to_numpy(), as:
plt.plot(outside.df['time'].to_numpy(),  outside.df['field'].to_numpy(), label='outside')

Maybe due to a difference in pandas version